### ISSUE TITLE
Crash on version 1.0.15.0

### ISSUE THREAD

**Issue by serialtasted** [jixxed/StationeersMods#11]
After updating to the newest version I'm getting this error and the game doesn't open

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
StationeersMods.Plugin.ModVersion.ToString () (at <cd01b34948af42e9984cca35be37389b>:0)
StationeersMods.ModManager+<>c.<validateModOrder>b__52_1 (StationeersMods.Plugin.ModVersion mod) (at <cd01b34948af42e9984cca35be37389b>:0)
HarmonyLib.GeneralExtensions+<>c__DisplayClass0_0`1[T].<Join>b__1 (System.String prev, T curr) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
System.Linq.Enumerable.Aggregate[TSource,TAccumulate] (System.Collections.Generic.IEnumerable`1[T] source, TAccumulate seed, System.Func`3[T1,T2,TResult] func) (at <b98680caec7f4227a10f36469340208c>:0)
HarmonyLib.GeneralExtensions.Join[T] (System.Collections.Generic.IEnumerable`1[T] enumeration, System.Func`2[T,TResult] converter, System.String delimiter) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
StationeersMods.ModManager.validateModOrder () (at <cd01b34948af42e9984cca35be37389b>:0)
StationeersMods.ModManager.WorldManagerFix () (at <cd01b34948af42e9984cca35be37389b>:0)
(wrapper dynamic-method) WorldManager.DMD<WorldManager::LoadDataFiles>()
WorldManager.LoadGameDataAsync () (at <144b446757384b10801c7e4c336406f9>:0)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTask`1[TStateMachine].GetResult (System.Int16 token) (at <4983a2b5b64a4bbcb8dbe527d7d390ed>:0)
WorldManager.Initialize () (at <144b446757384b10801c7e4c336406f9>:0)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTask`1[TStateMachine].GetResult (System.Int16 token) (at <4983a2b5b64a4bbcb8dbe527d7d390ed>:0)
Assets.Scripts.GameManager.Start () (at <144b446757384b10801c7e4c336406f9>:0)
System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) (at <7b935204f5ff4bcab44b3b0ebca330bf>:0)
UnityEngine.UnitySynchronizationContext+WorkRequest.Invoke () (at <e1f0f1bc777141aba161d8a739ac8634>:0)
UnityEngine.UnitySynchronizationContext.Exec () (at <e1f0f1bc777141aba161d8a739ac8634>:0)
UnityEngine.UnitySynchronizationContext.ExecuteTasks () (at <e1f0f1bc777141aba161d8a739ac8634>:0)
```

**Comment by jixxed** [comment]
do you happen to have a custom mod that is not released to the workshop?

**Comment by jixxed** [comment]
fixed in v1.0.16.0
